Makefile.am

```diff
@@ -92,6 +90,7 @@ rpm: local
 
 release: tag
 	$(MAKE) archive
+	$(MAKE) -C po/ $(PACKAGE_NAME).pot-update
```

Comment:
What is the point of this after the archive is created? I'm doing this as part of ``bumpver`` because I want the .pot file to have the new version in it.